### PR TITLE
Makes existing allergy quirk nicer, adds extreme subtype

### PR DESCRIPTION
**Pull request by atakiya**
## About The Pull Request
Adds
- insulin
- spaceacillin
- saline glucose
- coagulant

to the allergy quirk blacklist.

Renames that quirk to "Medicine Allergy", and adjusts its points

Adds an `/extreme` subtype  that inherits from the above and only leaves `insulin` and `saline glucose` as added blacklist items. Because its pretty much impossible to be allergic to those.

## Why It's Good For The Game
Adds a nice middle ground as an allergy perk, and also an extreme variant back incase someone really wants to die. Yet makes it a bit less awful as well.

## Changelog
:cl: Avunia Takiya
qol: Medicine Allergy Quirk now has insulin, spaceacillin, sal-glucose solution and coagulant in their blacklist - no more allergies from those
balance: Medicine Allergy Quirk only gives 4 points now
add: Re-added Extreme Medicine Allergy
balance: Extreme Medicine Allergy now has sal-glucose and insulin blacklisted
/:cl:

### CODE REVIEW

File: code/datums/quirks/negative.dm [hunk 1]
```diff
@@ -685,6 +697,25 @@
 			carbon_quirk_holder.vomit()
 			carbon_quirk_holder.adjustOrganLoss(pick(ORGAN_SLOT_BRAIN,ORGAN_SLOT_APPENDIX,ORGAN_SLOT_LUNGS,ORGAN_SLOT_HEART,ORGAN_SLOT_LIVER,ORGAN_SLOT_STOMACH),10)
 
+/datum/quirk/allergic/extreme
+	name = "Extreme Medicine Allergy"
+	desc = "Ever since you were born, you've been extremely allergic to certain chemicals..."
+	value = -6
+	gain_text = "<span class='danger'>You feel your immune system shift.</span>"
+	medical_record_text = "Patient's immune system responds violently to certain chemicals."
+	var/list/allergies = list()
```
Comment (Azarak) [inline code/datums/quirks/negative.dm:706]:
```suggestion
```

File: code/datums/quirks/negative.dm [hunk 2]
```diff
@@ -685,6 +697,25 @@
...
+	var/list/blacklist = list(
```
Comment (Azarak) [inline code/datums/quirks/negative.dm:707]:
```suggestion
	blacklist = list(
```

**Comment by Azarak** [comment]
What about taking both the allergy quirks?

**Comment by atakiya** [comment]
> What about taking both the allergy quirks?

Oops.

**Review by Azarak — APPROVED** [review]

**Pull request merged**